noir-projects/noir-contracts/contracts/schnorr_account_contract/src/main.nr

```diff
@@ -31,11 +31,15 @@ contract SchnorrAccount {
         let this = context.this_address();
         let header = context.get_header();
         let this_npk_m_hash = header.get_npk_m_hash(&mut context, this);
+        // Not emitting outgoing for msg_sender here to not have to register keys for the contract through which we
+        // deploy this (typically MultiCallEntrypoint). I think it's ok here as I feel the outgoing here is not that
+        // important.
+        let this_ovpk_m = header.get_ovpk_m(&mut context, this);
```

Comment:
Same as for the ecdsa account:

"I feel like given that it's a constructor it's commonly deployed from non contract address (I think it was throwing address 0 in the tests when I tried to change it) so makes sense to not have them set properly here."